client/resdef.go

```diff
@@ -22,19 +22,31 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/apis/batch"
+	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/client/restclient"
 )
 
+type ReplicaSet struct {
+	unversioned.TypeMeta `json:",inline"`
+
+	// Standard object metadata
+	v1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	ReplicaSet    *extensions.ReplicaSet `json:"replicaset"`
+	SuccessFactor int32                  `json:"success_factor"`
```

Comment:
As this is needed by https://github.com/Mirantis/k8s-AppController/pull/27/files#diff-9667cfa1ad6b8b445794f7c2469f069fR111 - we should keep this for now and as fast as possible do refactor in this areas.